modules/cbmarkdown/models/Processor.cfc

```diff
@@ -24,13 +15,13 @@ component accessors="true" singleton{
 	*/
 	function init( required javaLoader ){
 		// store references
-		variables.javaLoader = arguments.javaLoader;
-		// Setup markdown processor
-		variables.config = arguments.javaLoader.create( "com.github.rjeschke.txtmark.Configuration" )
-			.builder()
-			.forceExtentedProfile()
+		variables.jl = arguments.javaLoader;
+		variables.StaticParser = jl.create( "com.vladsch.flexmark.parser.Parser" );
```

Comment:
I have been using PascalCase for static java classes to distinguish them from instances of the class.  Thoughts?